cli/roger_push.py

```diff
@@ -210,76 +241,40 @@ def main(self, settings, appConfig, frameworkObject, hooksObj, args):
             if not hasattr(args, "app_name"):
                 args.app_name = ""
 
-            if 'registry' not in roger_env.keys():
-                raise ValueError("Registry not found in roger-mesos-tools.config file.")
-            else:
+            try:
                 self.registry = roger_env['registry']
+            except KeyError:
+                raise ValueError("Registry not found in roger-mesos-tools.config file.")
 
             if hasattr(args, "image_name"):
                 self.image_name = args.image_name
 
-            environment = roger_env.get('default_environment', '')
-            if args.env is None:
-                if "ROGER_ENV" in os.environ:
-                    env_var = os.environ.get('ROGER_ENV')
-                    if env_var.strip() == '':
-                        print(colored("WARNING - Environment variable $ROGER_ENV is not set. Using the default set from roger-mesos-tools.config file", "yellow"))
-                    else:
-                        if args.verbose:
-                            print(colored("Using value {} from environment variable $ROGER_ENV".format(env_var), "grey"))
-                        environment = env_var
-            else:
-                environment = args.env
-            # ----------------------------------------------
-
-            if environment not in roger_env['environments']:
-                raise ValueError("Environment not found in roger-mesos-tools.config file.")
-
+            environment = self.getTargetEnvironment(roger_env, args)
             # ----------------------------------------------
             # GetEnvironmentConfig(environment)
             # ----------------------------------------------
-            environmentObj = roger_env['environments'][environment]
-            common_repo = config.get('repo', '')
+            try:
+                environmentObj = roger_env['environments'][environment]
+            except KeyError as e:
```

Comment:
No reason to include "as e" if 'e' doesn't get used.

Reply:
It should be printed, made the change.